ksql-engine/src/main/java/io/confluent/ksql/function/udf/url/UrlDecodeKudf.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ **/
+
+package io.confluent.ksql.function.udf.url;
+
+import io.confluent.ksql.function.KsqlFunctionException;
+import io.confluent.ksql.function.udf.Kudf;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+
+
+public class UrlDecodeKudf implements Kudf {
```

Comment:
URL encoding/decoding is a complex business - This is a fairly generic name for a very precise implementation, i.e. decoding of query parameters (and form data).  How about `UrlQueryParamDecodeUdf`?